object/check.go

```diff
@@ -124,6 +124,18 @@ func CheckUserSignup(application *Application, organization *Organization, form
 		}
 	}
 
+	if application.IsSignupItemVisible("Invitation code") {
```

Comment:
logic is not correct. Only need to check `application.InvitationCodes`'s length > 0